src/components/TalentsListPage/SearchPanel/SearchPanel.jsx

```diff
@@ -0,0 +1,25 @@
+import { useMemo } from "react";
+import styles from "./SearchPanel.module.scss";
+//import st from "../../Layout/Header/Header.module.scss";
+export function SearchPanel() {
+    const inputNames = useMemo(
+        () => [
+            { name: "Job title" },
+            { name: "Location" },
+            { name: "Distance" },
+        ],
+        []
+    );
+    return (
+        <div className={styles["search-panel"]}>
+            {inputNames.map(({ name }) => (
+                <input
```

Comment:
I can recommend separating the input into a separate component and making it reusable, only then we can take out data about these inputs and use a loop to draw them